datafusion/common/src/display/mod.rs

```diff
@@ -62,6 +62,8 @@ pub enum PlanType {
     FinalPhysicalPlanWithStats,
     /// The final with schema, fully optimized physical plan which would be executed
     FinalPhysicalPlanWithSchema,
+    /// An error creating the physical plan
+    PhysicalPlanError,
```

Comment:
By adding an explicit new "plan type" it became clear how to show the errors

Reply:
Yes, that's what I want to do in the begining, but I didn't want to introduce many overhead so I changed it. Sometimes I am confused about  balancing change size and simplicity. Thanks for your suggestion!

Reply:
Yeah, it is totally a judgement call

Reply:
Yeah, it is totally a judgement call -- I don't think there is ever one right answer